test/unit/verify/test_models.py

```diff
@@ -41,3 +41,21 @@ def test_verification_materials_retrieves_rekor_entry(self, signing_materials):
         client = RekorClient.staging(tuf)
         entry = materials.rekor_entry(client)
         assert entry is not None
+
+    def test_rekor_entry_missing(self, signing_materials):
+        a_materials = signing_materials("a.txt")
+
+        # stub retriever post returning None RekorEntry
+        a_materials._offline_rekor_entry = None
+        client = pretend.stub(
+            log=pretend.stub(
+                entries=pretend.stub(
+                    retrieve=pretend.stub(
+                        post=pretend.call_recorder(lambda a, b, c: None)
```

Comment:
Small thing, but you don't have to use `call_recorder` if you aren't actually testing the callsites afterwards:

```suggestion
                        post=lambda a, b, c: None
```

Reply:
(Same with other unit tests, but don't worry too much about fixing them all up -- just something to be aware of in the future.)